config/irc-bot-external.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D Community Edition is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D Community Edition
+ *
+ * @author     HDVinnie <hdinnovations@protonmail.com>
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ */
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | IRC Bot External
+    |--------------------------------------------------------------------------
+    |
+    | IRC Bot External Settings
+    |
+    */
+
+    'is_enabled' => false,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Host
+    |--------------------------------------------------------------------------
+    |
+    | Host takes an ip/localhost as http endpoint. The api key is not required, but recommended.
```

Comment:
```suggestion
    | Host takes an ip/localhost as http endpoint.
```

It looks like the apikey is already described further down